grin/src/Grin/Parse.hs

```diff
@@ -1,140 +1,7 @@
-{-# LANGUAGE TupleSections, LambdaCase, OverloadedStrings #-}
+module Grin.Parse 
+  ( module Grin.ParseAST 
+  , module Grin.ParseTypeEnv 
```

Comment:
This could be in the a separate directory. `Grin\Parse\AST.hs` etc

Reply:
Done